app/user/rest.py

```diff
@@ -192,8 +192,10 @@ def verify_user_code(user_id):
         raise InvalidRequest("Code not found", status_code=404)
     if datetime.utcnow() > code.expiry_datetime or code.code_used:
         # sms and email
-        increment_failed_login_count(user_to_verify)
-        raise InvalidRequest("Code has expired", status_code=400)
+        # TODO: Fix email flow so that clicking link doesn't expire emails
+        if data['code_type'] != 'email':
```

Comment:
Can we just check of code is used? That way the link still expires in 30 minutes?

Reply:
good point. moved that check into the if statement above